app/frontendapi.py

```diff
@@ -0,0 +1,112 @@
+from django.http import JsonResponse
```

Comment:
Optional: Maybe better naming this file as frontend_api.py (with an underscore)?